src/components/Banner/Banner.stories.tsx

```diff
@@ -1,6 +1,6 @@
 import type { Meta, StoryObj } from '@storybook/react';
 import type { SyntheticEvent } from 'react';
-import { Banner } from './Banner';
+import { Banner } from '.';
```

Comment:
Thanks for reorganizing everything! Curious: Is there any advantage to renaming the components to `index.tsx`? I kind of prefer the more verbose `Component.tsx` filename because it makes it easier to find the file in my text editor.

Reply:
Brevity is probably the main win there.  `import { Banner } from './Banner/Banner.tsx'` felt excessive but I'm fine with prioritizing dev experience.  
 
I will update these names.